docs/scripts/formattedTSDemos.js

```diff
@@ -55,7 +55,8 @@ async function getFiles(root) {
           files.push(...(await getFiles(filePath)));
         } else if (
           stat.isFile() &&
-          /\.tsx?$/.test(filePath) &&
+          (/\.tsx?$/.test(filePath) || /\.ts?$/.test(filePath)) &&
```

Comment:
convert not just components, but other TS files, like `constants.ts`, `utils.ts` etc.

Reply:
Ah, you are right, I didn't notice the `?`, nice catch!